packages/taco/test/conditions/context.test.ts

```diff
@@ -122,7 +123,9 @@ describe('context', () => {
       const condition = new RpcCondition(testRpcConditionObj);
       const conditionContext = new ConditionContext(condition);
       expect(
-        JSON.stringify(condition.toObj()).includes(USER_ADDRESS_PARAM_DEFAULT),
+        JSON.stringify(condition.toObj(), toHexReplacer).includes(
```

Comment:
I think for here and in other places, anywhere that we are calling `JSON.stringify(...)` directly, should be replaced by the call to `toJSON(...)` from utils.